### PR TITLE
#6284 Why comment out the full connection layer for VGG16

### PR DESCRIPTION
**Pull request by crazymxm**
VGG16 needs full connection layer, so remove the comment out.

### CODE REVIEW

**Comment by crockpotveggies** [comment]
Going to wait for Jenkins to run a test on this and then I'll merge.

**Comment by saudet** [comment]
Looks like there is a build failure for the deeplearning4j-zoo module:
```
[ERROR] COMPILATION ERROR : 

[INFO] -------------------------------------------------------------

[ERROR] /home/jenkins/workspace/deeplearning4j-PR-6348-linux-x86_64-cpu/deeplearning4j/deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/model/VGG16.java:[161,50] package DenseLayer does not exist

[ERROR] /home/jenkins/workspace/deeplearning4j-PR-6348-linux-x86_64-cpu/deeplearning4j/deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/model/VGG16.java:[159,50] package DenseLayer does not exist

[INFO] 2 errors 
```
@crazymxm Please fix this!

**Comment by crazymxm** [comment]
Fix the compare error.

File: deeplearning4j/deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/model/VGG16.java [hunk 1]
```diff
@@ -181,7 +182,7 @@ public ComputationGraph init() {
 
     @Override
     public ModelMetaData metaData() {
-        return new ModelMetaData(new int[][] {inputShape}, 1, ZooType.CNN);
+        return null ;//new ModelMetaData(new int[][] {inputShape}, 1, ZooType.CNN);
```
Comment (saudet) [inline deeplearning4j/deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/model/VGG16.java]: Why is this returning null?

**Comment by saudet** [comment]
@crockpotveggies Good to merge?

**Comment by crockpotveggies** [comment]
Waiting for checks to pass and I'll merge it.

**Comment by sshepel** [comment]
@crockpotveggies The tests for `deeplearning4j-zoo` will not be run because there are some failures in tests for other modules :disappointed:

**Pull request merged**